core/src/main/scala/filodb.core/reprojector/MapDBMemTable.scala

```diff
@@ -56,119 +41,53 @@ class MapDBMemTable(config: Config) extends MemTable {
                           .cacheDisable
                           .make()
 
+  val serializer = new RowReaderSerializer(projection.columns)
+  val sortKeyFunc = projection.sortKeyFunc
+
+  implicit lazy val sortKeyOrdering: Ordering[K] = projection.helper.ordering
+  lazy val rowMap = db.createTreeMap("filo")
+                      .comparator(Ordering[(PartitionKey, K)])
+                      // Otherwise will pull out all values in a BTree Node at once
+                      .valuesOutsideNodesEnable()
+                      .counterEnable()
+                      .valueSerializer(Serializer.BYTE_ARRAY)
+                      .makeOrGet().asInstanceOf[BTreeMap[(PartitionKey, K), Array[Byte]]]
```

Comment:
Hrm, can we make makeOrGet accept a type param and have it maybe match on that internally and remove the need to clients calling it to have to do an InstanceOf reflection?